docs/developers/tutorials/eth-to-mvx.md

```diff
@@ -0,0 +1,385 @@
+
+Usually these gas costs for each blockchain are paid in the blockchain’s native currency.
+
+For Ethereum for instance, there is something called `gwei`, an unit equal to one-billionth of an ETH, or 10-9ETH. These fees have a tendency to grow depending on how crowded the network is. You can think of the gas cost as a result of a formula something like: `units of gas used * (base fee + priority fee)`, where the base fee differs per block, and will increase by a maximum of 12.5% per block if the target block size is exceeded. 
+
+The priority fee (tip) incentivizes validators to include a transaction in the block. Without tips, validators would find it economically viable to mine empty blocks, as they would receive the same block reward.
+
+For MultiversX, the gas used within the contract is terminologically separated from the unconsumed remaining gas. Here the processing time breaks the consumed gas  into two components:
+- gas used by `value movement and data handling`
+- gas used by `contract execution` (for executing System or User-Defined Smart Contract)
+
+The `value movement and data handling` component is defined by the formula:
+
+```
+tx.gasLimit = 
+    networkConfig.erd_min_gas_limit + networkConfig.erd_gas_per_data_byte * lengthOf(tx.data)
+```
+
+whereas the contract execution despite being easily computable for System Smart Contracts, it is harder to determine a priori for user-defined Smart Contracts. This is where simulations and estimations are employed.
+
+Finally, the processing fee used by the MultiversX blockchain looks something like this:
+
+```
+processing_fee =
+    value_movement_and_data_handling_cost * value_movement_and_data_handling_price_per_unit +
+    contract_execution_cost * contract_execution_price_per_unit
+```
+
+Let's look at some actual numbers!
+
+Based on https://etherscan.io/txs and https://beaconcha.in/charts/validators Ethereum at the moment of writing this tutorial had 1,135,937 transactions in the last 24h with an average transaction fee of 2.47 USD with 1,031,453 active validators.
+
+On the other side, MultiversX from their source (https://explorer.multiversx.com/) had 217,263 transactions in the last 24h with an average fee of 0.02 USD with 3,323 validators, which makes 100 times cheaper than a transaction on Ethereum.
+
+![img](/img/mvx-avg-tx.png)
+
+As you can see in the histogram above, the costs of the transactions on the MultiversX blockchain kept low. Note that the prices you see are in MultiversX’s native coin EGLD.
+
+![img](/img/mvx-max-fee.png)
+
+While sometimes when the Ethereum Network gets crowded the fee of a single transaction could easily jump over 47$, the fee of a single transaction on MultiversX in the last year barely reached 34.75$. Similar to the first histogram, the prices you see in the one with maximum transaction fee each month are in MultiversX’s native coin EGLD.
```

Comment:
this paragraph was formatted with a handwriting font, probably because of the `$` sign